site/docs/uk/hosting/vps.md

```diff
@@ -52,130 +152,631 @@ systemd --- це потужний менеджер служб, який пере
+   > Замініть `<токен-бота>` на токен вашого бота, а `<команда-запуску>` на команду, яку ви отримали [вище](#отримання-команди-запуску).
+
+   Ось коротке пояснення конфігурації сервісу:
+
+   - `After=network.target` --- вказує на те, що додаток повинен запускатися після завантаження інтернет-модуля.
+   - `Environment=BOT_TOKEN=<токен-бота>` --- встановлює змінну оточення `BOT_TOKEN`.
+     Додайте інші записи `Environment`, якщо вам потрібно декілька змінних оточення.
+   - `ExecStart=<команда-запуску>` --- встановлює команду запуску.
+   - `Restart=on-failure` --- вказує на те, що програма повинна перезапускатися після збою.
+   - `WantedBy=multi-user.target` --- визначає стан системи, в якому повинен запускатися сервіс.
+     `multi-user.target` --- типове значення для серверів, яке визначає стан системи, коли працює все, окрім графічного інтерфейсу, якого просто не існує.
+
+   > Для отримання додаткової інформації про файли служб, прочитайте [це] [цей сайт](https://access.redhat.com/documentation/te-in/red_hat_enterprise_linux/9/html/using_systemd_unit_files_to_customize_and_optimize_your_system/assembly_working-with-systemd-unit-files_working-with-systemd).
 
 4. Перезавантажуйте systemd кожного разу, коли ви редагуєте службу:
 
    ```sh
    systemctl daemon-reload
    ```
 
-### Управління службою
-
-#### Запуск
+#### Керування службою
 
 ```sh
+# Замініть `<назва-служби>` на назву файлу служби, яку ви створили.
+
+# Щоб запустити службу
 systemctl start <назва-служби>
+
+# Щоб переглянути журнали служби
+journalctl -u <назва-служби>
+
+# Щоб перезапустити службу
+systemctl restart <назва-служби>
+
+# Щоб зупинити службу
+systemctl stop <назва-служби>
+
+# Щоб увімкнути запуск служби при завантаженні сервера
+systemctl enable <назва-служби>
+
+# Щоб вимкнути запуск служби при завантаженні сервера
+systemctl disable <назва-служби>
+```
+
+### PM2
+
+[PM2](https://pm2.keymetrics.io/) — це демонічний менеджер процесів для Node.js, який допоможе вам керувати та тримати ваш застосунок в режимі онлайн 24/7.
+
+> PM2 створений спеціально для запуску застосунків, які написані на Node.js.
+> Проте його також можна використовувати для запуску застосунків, які написані на інших мовах або для інших фреймворків.
+
+#### Встановлення
+
+::: code-group
+
+```sh [NPM]
+npm install -g pm2
+```
+
+```sh [Yarn]
+yarn global add pm2
+```
+
+```sh [pnpm]
+pnpm add -g pm2
+```
+
+:::
+
+#### Створення застосунку
+
+PM2 пропонує два способи створення застосунку:
+
+1. За допомогою інтерфейсу командного рядка.
+2. За допомогою [файлу конфігурації](https://pm2.keymetrics.io/docs/usage/application-declaration/).
+
+Перший спосіб зручний під час знайомства з PM2.
+Проте під час розгортання варто скористатися другим способом, що ми й зробили у нашому випадку.
+
+На нашому сервері в каталозі, де зберігається збірка бота, розташовано файл `ecosystem.config.js` із наступним вмістом:
```

Comment:
```suggestion
На нашому сервері, в каталозі, де зберігається збірка бота, розташовано файл `ecosystem.config.js` із наступним вмістом:
```

Reply:
Wait, where did that file come from...?

Reply:
Well, users have to create it.

- [ ]

Reply:
@niusia-ua, Is there a need for a comma here? If not, just resolve this comment and that's it.

Reply:
I think that the comma is superfluous here